### DISCUSSION TITLE
Tabs at bottom of window covers bottom of page and status messages.

### DISCUSSION THREAD

**Discussion by dethzord** [Aris-t2/CustomCSSforFx#553] [General]
Not sure when it started happening, may have been during FF108, or maybe it has always been this way an I just noticed.  Running FF109 now and it is still present.

In any case, I use tabs_below_main_content.css.  The tab bar is covering up a small portion of the bottom of the page, as well as blocks the status/mouse hover popup.  Not terrible, but there have been times where I can't see links/buttons because of it.  If my description is unclear, I can throw up a screenshot.

There any fix for this?  Force the bar to be a part of the window rather than laid over the top of it?

**Comment by Aris-t2** [comment]
Make sure you are using this projects latest release. Some major changes were done for Fx109 in the last few builds.
Also post a screenshot of the issue, after you "installed" and setup v4.3.0 release.

**Comment by dethzord** [comment]
Thanks, didn't realize so much had changed.  Updated, and the issue is resolved.  Now I just need to figure out all the changes I made; it was a while ago.

Apologies for not first updating.  Should have done so before posting.
